ftw/django_utils/settings/logging.py

```diff
@@ -0,0 +1,100 @@
+import sys
+
+import sentry_sdk
+from configurations import values
+from sentry_sdk.integrations.django import DjangoIntegration
+
+
+class LogMixin:
+    """
+    This mixins contains the logging configuration of our Django based
+    web applications. Its default values are suited for production.
+    """
+
+    @property
+    def LOGGING_FILENAME(self):
```

Comment:
I was wondering if we ever configured the log file through an environment variable or if we could ditch this?

Reply:
For the sake of consistency, I'd remove this. This will enforce the same log files in all of our projects.